### DISCUSSION TITLE
android 版本SDK如何支持beta版特性？

### DISCUSSION THREAD

**Discussion by GerrieWell** [Tencent/libpag#2275] [Q&A] [answered]
我有一份ae模型，需要设置beta版才能正常导出，在电脑版pagViewer中正常预览。
但是我androidSDK加载该pag 文件无法预览，最新4.2.115和4.3.51都是，请问androidSDK如何加载beta版导出的pag文件？

**Comment by kevingpqi123** [comment, marked as answer]
需要看的应该是这里：https://pag.art/zh-CN/feature.html
是否使用了企业版的独有特性，上面的链接中罗列了企业版和社区版的能力差异
